lib/sourceror.ex

```diff
@@ -199,8 +199,14 @@ defmodule Sourceror do
     extract_comments_opts = [collapse_comments: true, correct_lines: true] ++ opts
 
     {quoted, comments} = Sourceror.Comments.extract_comments(quoted, extract_comments_opts)
+    locals_without_parens = Keyword.get(opts, :locals_without_parens, locals_without_parens())
 
-    to_algebra_opts = Keyword.merge(opts, comments: comments, escape: false)
+    to_algebra_opts =
+      Keyword.merge(opts,
+        comments: comments,
+        escape: false,
+        locals_without_parens: locals_without_parens
+      )
```

Comment:
I would be fine if we do not overwrite `locals_without_parens`.
```suggestion
    to_algebra_opts =
      opts
      |> Keyword.merge(comments: comments, escape: false)
      |> Keyword.put_new(:locals_without_parens, locals_without_parens())
```

Reply:
I've considered this, but if we don't provide the override feature, will it be a problem in the future when `sourceror` is relied on by some tools like elixir_ls?

I mean `elixirls` is a separate project, but it may really need is  the `locals_without_parens` from the user project, not 
https://github.com/elixir-lsp/elixir-ls/blob/master/.formatter.exs

Reply:
This change would let the caller of `Sourceror.to_string/2` decide what `:local_without_parens` are used. If someone does not set `:local_without_parens` the values of the current mix project will be used. If someone wants to set `:local_without_parens` he can do it (for whatever reason).

Reply:
Sorry, I missed the `put_new/3`. `put_new/3` is the same as what I wanted